api/models/bidDao.js

```diff
@@ -1,20 +1,230 @@
-const dataSource = require("./dataSource");
+const appDataSource = require("./dataSource");
 
-const getBidById = async (bidId) => {
-  return await dataSource.query(
+const bidStatus = Object.freeze({
+  "입찰전": 1,
+  "입찰중": 2,
+  "낙찰완료": 3,
+  "경매종료": 4
+})
+
+const isBidEnd = async (itemId) => {
+  const now = new Date();
+  const utc = now.getTime() + (now.getTimezoneOffset() * 60 * 1000);
+  const KR_TIME_DIFF = 9 * 60 * 60 * 1000;
+  const kr_now = new Date(utc + (KR_TIME_DIFF))
+
+  const [biddingEnd] =
+    await appDataSource.query(`
+        SELECT bidding_end
+        FROM items
+        WHERE id = ?
+    `, [itemId])
+  const [biddingStart] = await appDataSource.query(`
+      SELECT bidding_start
+      FROM items
+      WHERE id = ?
+  `, [itemId])
+
+  const endDate = new Date(biddingEnd["bidding_end"])
+  const endDateNow = new Date(endDate.getTime() + KR_TIME_DIFF)
+
+  const startDate = new Date(biddingStart["bidding_start"])
+  const startDateNow = new Date(startDate.getTime() + KR_TIME_DIFF)
+
+  let result = ""
+  if (kr_now < startDateNow) {
+    result = "입찰전"
+  } else if (kr_now >= endDateNow && (await isBidExist(itemId)) === true) {
+    result = "경매종료"
+  } else if (kr_now >= endDateNow && (await isBidExist(itemId)) === false) {
+    result = "낙찰완료"
+  } else if (kr_now < endDateNow || kr_now >= startDateNow) {
+    result = "입찰중"
+  }
+  return result
+}
+
+const isPassedOneDayAfterBiddingEnd = async (itemId) => {
+  const now = new Date();
+  const utc = now.getTime() + (now.getTimezoneOffset() * 60 * 1000);
+  const KR_TIME_DIFF = 9 * 60 * 60 * 1000;
+  const kr_now = new Date(utc + (KR_TIME_DIFF))
+
+  const [biddingEnd] =
+    await appDataSource.query(`
+        SELECT bidding_end
+        FROM items
+        WHERE id = ?
+    `, [itemId])
+
+  const endDate = new Date(biddingEnd["bidding_end"])
+  const oneDayAfterEndDate = new Date(endDate.getTime() + KR_TIME_DIFF + 24 * 60 * 60 * 1000)
+
+  if (oneDayAfterEndDate < kr_now) {
+    return true
+  }
+  return false
+}
```

Comment:
```suggestion
const isPassedOneDayAfterBiddingEnd = async (itemId) => {
  const now = getKoreanTime(new Date());

  const [biddingEnd] = await appDataSource.query(`
    SELECT bidding_end
    FROM items
    WHERE id = ?
  `, [itemId]);

  const endDate = new Date(biddingEnd["bidding_end"]);
  const oneDayAfterEndDate = new Date(endDate.getTime() + 24 * 60 * 60 * 1000);

  return oneDayAfterEndDate < now;
};
```
위 내용에 이어서 남겨드립니다. 모듈화한 함수를 이용하고 로직을 간단히 할 수 있고, return에 들어가는 if문은 없어도 되는 코드입니다! `return oneDayAfterEndDate < now;` 코드 자체가 경우에 따라 참거짓을 반환합니다.